content/docs/gherkin/reference.md

```diff
@@ -260,6 +260,32 @@ Example: Multiple Givens
   But I shouldn't see something else
 ```
 
+### Star Notation
+
+Gherkin also supports a "star" notation for steps, where the `*` character can be used in place of any of the normal step keywords. This can helpful when you have some steps that are effectively a _list of things_, so you can express it more like bullet points where otherwise the natural language of `And` etc might not read so elegantly.
+
+For example:
+
+```gherkin
+Example: List of setup items
```

Comment:
Lets use an example here of a shopping list (As above uses concretions).

So you could say

```
Given I have eggs
And I have milk
And I have butter
....
```

```
Given I have eggs
* I have butter
* I have milk
```

Reply:
Oh yeah, that could work. To expand on it a bit:

```gherkin
Scenario: All done
  Given I am out shopping
  * I have eggs
  * I have milk
  * I have butter
  When I check my list
  Then I don't need anything
```

I feel like you can see clearly how that reads better than `And`s. What do you think?

Reply:
Updated

Reply:
I’m not sure I like the doc text here, because to me, `And` reads more naturally than a sort of half-bullet, half-sentence syntax. Generally if I want something to look like bullet points, I’ll put a list in a data table so I don’t need to have this awkward structure...

However, I know we’re not discussing the feature but only its documentation. I just strongly disagree with the idea that the stars are an improvement in the given case.